scripts/gitlab/build_and_test.sh

```diff
@@ -30,6 +30,11 @@ spack_debug=${SPACK_DEBUG:-false}
 debug_mode=${DEBUG_MODE:-false}
 push_to_registry=${PUSH_TO_REGISTRY:-true}
 perf_artifact_dir=${PERF_ARTIFACT_DIR:-""}
+performance_tests=${PERFORMANCE_TESTS:-false}
+kernels=${KERNELS:-""}
+variants=${VARIANTS:-""}
```

Comment:
Variant is overloaded, I checked whether it was a spack variant, but it's a kernel variant, or a perf_variant.
Same in the CI config: VARIANT coexists with SPEC, which is a bit ambiguous. I’m tempted to suggest PERF_ prefix. Would it be to heavy ?